rpcs3/Crypto/unself.cpp

```diff
@@ -1322,9 +1322,7 @@ bool SELFDecrypter::GetKeyFromRap(u8* content_id, u8* npdrm_key)
 
 	if (!rap_file)
 	{
-		self_log.fatal("Failed to locate the game license file: %s."
-				  "\nEnsure the .rap license file is placed in the dev_hdd0/home/00000001/exdata folder with a lowercase file extension."
-				  "\nIf you need assistance on dumping the license file from your PS3, read our quickstart guide: https://rpcs3.net/quickstart", rap_path);
+		self_log.fatal("Failed to load RAP file: %s. Make sure the file extension is lowercase.", rap_path);
```

Comment:
Why this weird revert

Reply:
The link is no longer needed as it's in the dialog box, and I dislike having such a long and lenghty message printed to log, I can drop the revert if you want to keep it around, but it's ugly and not neccessary.

Reply:
but you are removing information that was added by helpers in order to get less spam